tests/llm/conftest.py

```
@@ -221,45 +242,172 @@ def force_pytest_output(request):
+            error_messages.append(provider_msg)
+            continue  # Skip API test if env vars are missing
+
+        # Step 2: Environment is OK, now test if the API actually works
+        try:
+            resp = litellm.completion(
+                model=model_name,
+                messages=[{"role": "user", "content": "test"}],
+                max_tokens=1000,
+            )
+            print(resp)
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)